plonky_block_proof_gen/src/lib.rs

```diff
@@ -81,18 +83,14 @@
 //! both statement into one, effectively proving an entire chain from genesis
 //! through a single final proof.
 //!
-//! ```rust
+//! ```compile_fail
```

Comment:
Nit: wouldn't ignore be more appropriate here? Just wondering what's more rust-like

Reply:
Hmm... We could switch to `ignore`, but my thought was we would at least get the syntax checking if we use this option instead. This way if the API changes then the docs will still pick this up.

Reply:
Ah good point, let's keep it as is then :)